src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -453,78 +446,116 @@ const HotspotDetails = ({
           <Box
             justifyContent="flex-start"
             flexDirection="row"
-            backgroundColor="grayBox"
+            backgroundColor="grayBoxLight"
           >
             <HeliumSelect
               showGradient={false}
               marginTop="m"
               contentContainerStyle={contentContainerStyle}
               data={selectData}
-              backgroundColor="grayBox"
+              backgroundColor="grayBoxLight"
               selectedValue={selectedOption}
               onValueChanged={handleSelectValueChanged}
             />
           </Box>
           <HotspotChecklist
             paddingTop="lx"
```

Comment:
I think the `<AutoPill />`'s in here need to have a darker background. Maybe `grayPurple` instead of `grayBox`
![3 4 0 2021-07-12 at 11 17 46 AM](https://user-images.githubusercontent.com/1760821/125322134-0ac4aa80-e303-11eb-847d-7ad782a4636a.jpg)